Makefile

```diff
@@ -0,0 +1,39 @@
+# several pytest settings
+DURATION ?= all  # pytest duration, one of short, long or all
+WORLD_SIZE ?= 1  # world size for DDP tests
+EXTRA_ARGS ?=  # additional arguments
+
+EXTRA_ARGS := --duration $(DURATION) $(EXTRA_ARGS)
```

Comment:
Can we add `--tb=short --disable-pytest-warnings` also? Test failures create super long and hard-to-localize traces without `--tb=short`. For warnings, almost all of them are from deep in the bowels of third-party libraries. It would in theory be good practice to read them, but it's such a wall of 95% useless text that I doubt anyone will (I almost never do).

Reply:
I'm going to default to the same verbosity as `pytest` itself. However, users can adjust that directly, e.g.:
```
make test EXTRA_ARGS=--tb-short --disable-pytest-warnings
```

We've already filtered out most of the warnings that can be safely ignored (see pyproject.toml), so there shouldn't be as many now.